tests/utils.go

```diff
@@ -4330,7 +4330,11 @@ func resetToDefaultConfig() {
 	}
 
 	UpdateKubeVirtConfigValueAndWait(KubeVirtDefaultConfig)
-	UpdateCDIConfigMap(CDIInsecureRegistryConfig)
+
+	// On SRIOV we dont have Insecure registries
```

Comment:
Please change the reason to cdi operator is not deployed on sriov-lane

Reply:
imo its not needed, since this function mess with the reg, not with the cdi

Reply:
Actually we do have insecure registry (the local registry) it is configured here https://github.com/kubevirt/kubevirt/blob/master/cluster-up/cluster/kind/common.sh#L207
I asked to change this comment because it does not describe the correct reason to skip `UpdateCDIConfigMap`.

Reply:
i will add then the word CDI insecure, this was of course the meaning, 
as the function checks the CDI insecure registries, the comment main reason was SRIOV, not which type of insecure registries those are (which can be read from the code itself)

Reply:
Done

Reply:
added also a note to the PR description, reflecting what i think, combined with what you ask.

Reply:
I dont this we should comment that we dont have CDI insecure registries, we should comment that we dont have CDI at all.
One could think that we just dont use insecure registries and CDI is there.

Reply:
lets hear please then other opinions

Reply:
i removed the comment

Reply:
I prefer you to keep the comment, there is no documentation for CDI not being deployed on this level (only if you dig in until you get into cluster-deploy.sh which run before this code runs)

Reply:
I am against comments, unless they will allow me to move forward :)
Edy please suggest, we dont have an agreement here and i think we said all already

Reply:
@ormergi what comment you want exactly? The condition should reflect that CDI is not deployed, what else you want to add to that?

Reply:
At the current state the code does not reflect that we do not deploy cdi on sriov lane, thus I would expect comment just defore the `if` : `CDI opertator does not deployed on SRION lane`

Reply:
We should not care why CDI is not deployed... these tests may run against a non kibevirtci cluster and such a comment will be just wrong.

In general, comments will eventually rot because no one will update them if the env changes.
Your last resort should be to use comments.